release-notes/opensearch-security.release-notes-3.0.0.0-beta1.md

```diff
@@ -0,0 +1,57 @@
+## Version 3.0.0-beta1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards version 3.0.0-beta1
+
+#### Breaking Changes
+* Optimized Privilege Evaluation ([#4380](https://github.com/opensearch-project/security/pull/4380))
```

Comment:
I am wondering whether we should really call this a breaking change. In the end, it should be not breaking except of the discontinuation of the undocumented option `multi_rolespan_enabled`, which is described in its own issue at https://github.com/opensearch-project/security/issues/4495

Reply:
Good catch. I copied this from the release notes from alpha1. Changed the category to Enhancement.